File: C++/1035.cpp
#include <iostream>
 
using namespace std;
 
int main() {
    int a,b,c,d;
    cin >> a >> b >> c >> d;
    if (b > c && d > a && (c+d) > (a+b) && d > 0 && c>0){
        cout << "Valores aceitos" << endl;
    } else {
        cout << "Valores nao aceitos\n" << endl;
    }
    
    return 0;
}

File: Ruby/1035.rb
a,b,c,d = gets.split.map(&:to_i)

def selectTest(a,b,c,d)
	if a.even? == true and d > a and b > c and (c+d) > (a+b) and d > 0 and c > 0
			return "Valores aceitos"
	else
		return "Valores nao aceitos"
	end
end

puts selectTest(a,b,c,d)
